src/xmipp/applications/scripts/deep_micrograph_cleaner/deep_micrograph_cleaner.py

```diff
@@ -72,30 +73,22 @@ def defineParams(self):
         self.addExampleLine('xmipp_deep_micrograph_cleaner -c path/to/inputCoords/ -o path/to/outputCoords -b $BOX_SIXE  -i  /path/to/micrographs/')
         
     def run(self):
-
-      try:
-        from xmippPyModules.deepLearningToolkitUtils.utils import checkIf_tf_keras_installed, updateEnviron
-      except ImportError:
-        print("Import error")
-        print(sys.path)
-        from pylib.xmippPyModules.deepLearningToolkitUtils.utils import checkIf_tf_keras_installed, updateEnviron
-
-        checkIf_tf_keras_installed()
+        # checkIf_tf_keras_installed()
         args={}
         gpusToUse="0"
         if self.checkParam('-g'):
           gpusToUse= self.getParam('-g')
           if "None" in gpusToUse or "-1" in gpusToUse:
-            gpusToUse=None
+            gpusToUse=-1
         args["gpus"]=gpusToUse
 
-        updateEnviron(gpusToUse)
-        
         if self.checkParam('-i'):
-          mdObj= md.MetaData(os.path.expanduser( self.getParam('-i')))
+          mdObj= xmippLib.MetaData(os.path.expanduser( self.getParam('-i')))
+
           args["inputMicsPath"]= []
           for objId in mdObj:
-            args["inputMicsPath"]+= [mdObj.getValue(md.MDL_IMAGE, objId)]
+            args["inputMicsPath"]+= [mdObj.getValue(xmippLib.MDL_IMAGE, objId)]
+          args["inputMicsPath"]= " ".join(args["inputMicsPath"])
```

Comment:
I'm not big fan of changing the type of the variable. Here you got from list to string, which is at least confusing

Reply:
they ultimately become command line arguments and thus, they need to be strings.